hack/make-rules/test-cmd.sh

```diff
@@ -73,7 +71,7 @@ function run_kube_apiserver() {
     --token-auth-file=hack/testdata/auth-tokens.csv 1>&2 &
   export APISERVER_PID=$!
 
-  kube::util::wait_for_url "http://127.0.0.1:${API_PORT}/healthz" "apiserver"
+  kube::util::wait_for_url "https://127.0.0.1:${SECURE_API_PORT}/healthz" "apiserver" 1 30 1 '--oauth2-bearer admin-token'
```

Comment:
changing `AUTHORIZATION_MODE` to `"RBAC,Node"` would enable anonymous auth and allow healthz to be fetched without passing these extra args

Reply:
If we changed `AUTHORIZATION_MODE` from `RBAC,AlwaysAllow` to `RBAC,Node`, the test [`run_impersonation_tests`](https://github.com/kubernetes/kubernetes/blob/518b826b1dcc4852226300ae08110cba03a078c9/test/cmd/authorization.sh#L56) would not pass, because it needs to run `kubectl create ... --as=user1`.

I could think of 2 solutions:
1. leave `AUTHORIZATION_MODE` intact, and add a new util function `kube::util::wait_for_url_with_bearer_token` which accesses the url with given token.
2. change `AUTHORIZATION_MODE` to `RBAC,Node`, and add a new token whose user name is `user1` to https://github.com/kubernetes/kubernetes/blob/master/hack/testdata/auth-tokens.csv

I decided to use 1) here, WDYT?